package assignments;

public class Assignment_no41 {
	public static void main(String[] args) {
		
		try {
			int a=100/0;
		}
		catch(ArithmeticException e) {
			System.out.println("exception occures");
		}
		
	}

}
